src/static/chemiscope.css

```diff
@@ -160,6 +160,10 @@
     width: auto;
 }
 
+#chsp-color-reset {
```

Comment:
instead of doing this (which will break when the map get a unique id in #132), you should add this bit of styling in the HTML directly